.github/workflows/beam_LoadTests_Go_Combine_Flink_Batch.yml

```diff
@@ -0,0 +1,133 @@
+  security-events: read
+  statuses: read
+
+# This allows a subsequently queued workflow run to interrupt previous runs
+concurrency:
+  group: '${{ github.workflow }} @ ${{ github.event.issue.number || github.sha || github.head_ref || github.ref }}-${{ github.event.schedule || github.event.comment.body || github.event.sender.login }}'
+  cancel-in-progress: true
+
+env:
+  GRADLE_ENTERPRISE_ACCESS_KEY: ${{ secrets.GE_ACCESS_TOKEN }}
+  GRADLE_ENTERPRISE_CACHE_USERNAME: ${{ secrets.GE_CACHE_USERNAME }}
+  GRADLE_ENTERPRISE_CACHE_PASSWORD: ${{ secrets.GE_CACHE_PASSWORD }}
+  GCLOUD_ZONE: us-central1-a
+  CLUSTER_NAME: beam-loadtests-go-combine-flink-batch-${{ github.run_id }}
+  GCS_BUCKET: gs://beam-flink-cluster
+  FLINK_DOWNLOAD_URL: https://archive.apache.org/dist/flink/flink-1.15.0/flink-1.15.0-bin-scala_2.12.tgz
+  HADOOP_DOWNLOAD_URL: https://repo.maven.apache.org/maven2/org/apache/flink/flink-shaded-hadoop-2-uber/2.8.3-10.0/flink-shaded-hadoop-2-uber-2.8.3-10.0.jar
+  FLINK_TASKMANAGER_SLOTS: 1
+  DETACHED_MODE: true
+  HARNESS_IMAGES_TO_PULL: gcr.io/apache-beam-testing/beam-sdk/beam_go_sdk:latest
+  JOB_SERVER_IMAGE: gcr.io/apache-beam-testing/beam_portability/beam_flink1.15_job_server:latest
+  ARTIFACTS_DIR: gs://beam-flink-cluster/beam-loadtests-go-combine-flink-batch-${{ github.run_id }}
+
+jobs:
+  beam_LoadTests_Go_Combine_Flink_Batch:
+    if: |
+      github.event_name == 'workflow_dispatch' ||
+      github.event_name == 'schedule' ||
+      github.event.comment.body == 'Run Load Tests Go Combine Flink Batch'
+    runs-on: [self-hosted, ubuntu-20.04, main]
+    timeout-minutes: 720
+    name: ${{ matrix.job_name }} (${{ matrix.job_phrase }})
+    strategy:
+      matrix:
+        job_name: ["beam_LoadTests_Go_Combine_Flink_Batch"]
+        job_phrase: ["Run Load Tests Go Combine Flink Batch"]
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup repository
+        uses: ./.github/actions/setup-action
+        with:
+          comment_phrase: ${{ matrix.job_phrase }}
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          github_job: ${{ matrix.job_name }} (${{ matrix.job_phrase }})
+      - name: Prepare test arguments
+        uses: ./.github/actions/test-arguments-action
+        with:
+          test-type: load
+          test-language: go
+          argument-file-paths: |
+            ${{ github.workspace }}/.github/workflows/load-tests-job-configs/config_Combine_Go_Batch_Flink_10b.txt
+            ${{ github.workspace }}/.github/workflows/load-tests-job-configs/config_Combine_Go_Batch_Flink_Fanout_4.txt
+            ${{ github.workspace }}/.github/workflows/load-tests-job-configs/config_Combine_Go_Batch_Flink_Fanout_8.txt
+          arguments: |
+            --job_name=load-tests-go-flink-batch-combine-$(date '+%m%d%H%M%S' --utc)
+      - name: Start Flink with parallelism 5
+        env:
+          FLINK_NUM_WORKERS: 5
+        run: |
+          cd ./.test-infra/dataproc; ./flink_cluster.sh create
```

Comment:
Can we run `.${{ github.workspace }}/.test-infra/dataproc/flink_cluster.sh create` instead of drilling down to the dataproc directory first? The same comment for the `- name: Restart Flink with parallelism 16` step

Reply:
No, we should change directory, because there are relative paths inside this script